src/components/board.jsx

```diff
@@ -21,6 +21,7 @@ function Board() {
     <div className="container mt-4">
       {/* <h3 className="text_center">CLIPNOW</h3> */}
       <h5 className="text_center">Create your CLIP!</h5>
+      <p>Note: Only Alphanumerics are allowed</p>
```

Comment:
@ChsPranav, Please remove the 'p' tag statement, as the ID validation message will be shown by the browser itself.